code/modules/mob/living/silicon/ai/ai.dm

```diff
@@ -435,6 +438,7 @@ var/list/ai_verbs_default = list(
 /mob/living/silicon/ai/emp_act(severity)
 	if (prob(30))
 		view_core()
+	icon_state = "ai-fuzz"

```

Comment:
Ya, it ain't permanent. It's also kind of a bitch to reset by code, unless you write a spawn() command I guess.
```
    var/old_state = icon_state
    icon_state = "ai-fuzz"
    spawn (n)
        if (icon_state == "ai-fuzz")
            icon_state = old_state
```